src/database/models/base.py

```
@@ -0,0 +1,7 @@
+from sqlalchemy.orm import DeclarativeBase
+
+
+class Base(DeclarativeBase):
```

Pylint (3.3.7)

[convention] Missing class docstring

(C0115)